From: Simon McVittie <smcv@collabora.com>
Date: Tue, 29 Aug 2017 17:05:16 +0100
Subject: Remove shebang from bash completions

bash completions are to be sourced. It makes little sense to
execute them.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 bash/ostree | 2 --
 1 file changed, 2 deletions(-)

diff --git a/bash/ostree b/bash/ostree
index f4305f6..da8b9af 100644
--- a/bash/ostree
+++ b/bash/ostree
@@ -1,5 +1,3 @@
-#!/bin/bash
-#
 # bash completion file for ostree commands
 #
 # This script provides completion of:
